sdk/src/main/scala/io/horizen/account/state/AccountState.scala

```diff
@@ -210,7 +210,11 @@ class AccountState(
       stateView.updateForgerBlockCounter(mod.forgerPublicKey, consensusEpochNumber)
 
       // If SC block has reached the end of the withdrawal epoch reward the forgers.
-      evalForgersReward(mod, modWithdrawalEpochInfo, consensusEpochNumber, stateView)
+      val distributionCap = if (Version1_4_0Fork.get(consensusEpochNumber).active) {
```

Comment:
This value is in Satoshi, aka Zennies, shouldn't we consider wei value instead?

Reply:
Thanks! Totally missed that